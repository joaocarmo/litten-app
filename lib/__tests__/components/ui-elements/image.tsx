import { UIImage } from '@ui-elements'
import TestRenderer from 'react-test-renderer'

const imgPlaceholder = 'https://placeimg.com/140/140/people'
describe('Snapshot test for the "UIImage" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <UIImage source={imgPlaceholder} />,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
