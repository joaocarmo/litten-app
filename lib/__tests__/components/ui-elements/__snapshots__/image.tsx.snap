// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UIImage" component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Array [
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ],
    ]
  }
>
  <FastImageView
    resizeMode="contain"
    source={
      Object {
        "uri": "https://placeimg.com/140/140/people",
      }
    }
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
